lib/Config.js

```diff
@@ -75,6 +75,29 @@ class Config {
             }
         }
 
+        this.cdmi = { };
+        if (config.cdmi !== undefined) {
+            if (config.cdmi.host !== undefined) {
+                assert.strictEqual(typeof config.cdmi.host, 'string',
+                                   'bad config: cdmi host must be a string');
+                this.cdmi.host = config.cdmi.host;
+            }
+            if (config.cdmi.port !== undefined) {
+                assert(Number.isInteger(config.cdmi.port)
+                       && config.cdmi.port > 0,
+                       'bad config: cdmi port must be a positive integer');
+                this.cdmi.port = config.cdmi.port;
+            }
+            if (config.cdmi.path !== undefined) {
+                assert(typeof config.cdmi.path === 'string',
+                       'bad config: cdmi.path must be a string')
+                assert(config.cdmi.path.length > 0,
+                       'bad config: cdmi.path is empty');
+                assert(config.cdmi.path.charAt(0) === '/',
+                       'bad config: cdmi.path should start with a \'/\'');
```

Comment:
it would be clearer to use double quotes inside the single quotes rather than escaping.
```javascript
'with a "/\"'
```